src/django/api/constants.py

```diff
@@ -147,3 +147,22 @@ class DateFormats:
     SECOND = '%Y-%m-%d %H:%M:%S'
     MONTH = '%Y-%m'
     WEEK = '%Y-%W'
+
+
+class NumberOfWorkersRanges:
+    STANDARD_RANGES = [{
+        'min': 0,
+        'max': 1000,
+        'text': 'Less than 1000'
```

Comment:
Minor, but consider renaming this property `label`, which I think makes the usage [here](https://github.com/open-apparel-registry/open-apparel-registry/pull/1651/files#diff-900f6cb047297d79255398d8083917fe71bcfc43e15e933ef8a322787fcf5508R103) a bit more clear.

Reply:
Good call. I've updated the property name.